### ISSUE TITLE
Two and more interval for severity

### ISSUE THREAD

**Issue by Nikolay-Ch** [radimkeseg/lovelace-mini-gauge-card#2]
Ex, if I want to show voltage value, I need to show it in green for values between 200 and 250, yellow for values from 180 to 200 and from 250 to 270, and red for values lower than 180 or higher than 270...

**Comment by radimkeseg** [comment]
interesting use case, how would you expect to the cofig yaml shold look like?
e.g.:
```YAML
severity:
  red: -1000
  yellow: 180
  green: 200
  yellow: 250
  red: 270
```

would you please suggest your idea?

**Comment by Nikolay-Ch** [comment]
Maybe like intervals and color for interval? LIke in "Canvas Gauge Card"...

```YAML
highlights: [{ "from": 80, "to": 100, "color": "rgba(200, 50, 50, .75)" }, { "from": 101, "to": 150, "color": "yellow" }]
```

**Comment by radimkeseg** [comment]
ok, I have decided to implement it slightly differently, however you should be able to achieve what you need
exmaple of config could  would read following way:

```YAML
min: 0
max: 100
severity:
  red_L: 10
  yellow_L: 20
  green: 40
  yellow_H: 60
  red_H: 80
```


will commit soon

**Comment by radimkeseg** [comment]
implemented in 0.2.6
